src/core/server/elasticsearch/elasticsearch_config.ts

```diff
@@ -298,6 +299,11 @@ export class ElasticsearchConfig {
    */
   public readonly apiVersion: string;
 
+  /**
+   * The maximum number of sockets that can be used for communications with elasticsearch.
+   */
+  public readonly maxSockets?: number;
```

Comment:
I think the `defaultValue` declared in the schema makes this property always available. We may define it as required